### DISCUSSION TITLE
Yeni Katılanlar İçin Eğitim Videosu İzleme [Zorunlu]

### DISCUSSION THREAD

**Discussion by Ciklamert** [gelecekbilimde/backend#157] [Announcements]
Ekibe yeni katılanların, proje süreçlerine daha hızlı adapte olabilmesi için eğitim videosunu izlemeleri gerekmektedir: 
	• Videoyu izledikten sonra, lütfen bu task altına yorum bırakarak bildirin.
	• Bu bildirim, task almaya hazır hale geldiğiniz anlamına gelecektir. Bildirmeyen kişiler task almayacaktır.

https://www.youtube.com/watch?v=RmzNWGhIayE

**Comment by sedefbas** [comment]
izledim.

**Comment by ahmetbeyaz** [comment]
izledim

**Comment by MenekseYuncu** [comment]
izledim

**Comment by hnurr** [comment]
izledim.

**Comment by omerbirgul** [comment]
izledim

**Comment by zehrayhn** [comment]
izledim.
